airflow/www_rbac/templates/airflow/dags.html

```diff
@@ -235,11 +235,32 @@ <h2>DAGs</h2>
       const DAGS_INDEX = "{{ url_for('Airflow.index') }}";
       const ENTER_KEY_CODE = 13;
 
+      function createOrAppendToQueryString(parameter, value) {
+        var query_string_parameters = {};
+        if (typeof window.location.search !== "undefined") {
+          window.location.search.substr(1).split('&').forEach(function (query_string_parameter) {
```

Comment:
This code is repeated. In this case, you should pull the shared part into a separate file and add the link. Otherwise the code creates unnecessary confusion and mess.

Reply:
In this case it is okay - the non "rbac" version will be deleted from the tree soon-ish.